File: test/fixtures/safe/crypto_usage.js
const crypto = require('crypto');

// AES-256-GCM - QUANTUM SAFE
const key = crypto.randomBytes(32);
const cipher = crypto.createCipheriv('aes-256-gcm', key, iv);

// SHA-256 - QUANTUM SAFE
const hash = crypto.createHash('sha256');

File: test/fixtures/safe/crypto_usage.py
from cryptography.hazmat.primitives.ciphers.aead import AESGCM
import hashlib

# AES-256-GCM - QUANTUM SAFE
key = AESGCM.generate_key(bit_length=256)
aesgcm = AESGCM(key)

# SHA-256 - QUANTUM SAFE
digest = hashlib.sha256(b"data").hexdigest()

# SHA-3 - QUANTUM SAFE
sha3_hash = hashlib.sha3_256(b"data").hexdigest()
